ports/stm/Makefile

```diff
@@ -128,66 +134,72 @@ ifndef BOOTLOADER_OFFSET
 	BOOTLOADER_OFFSET := 0x8000000
 endif
 
-LDFLAGS = $(CFLAGS) -fshort-enums -Wl,-nostdlib -Wl,-T,$(LD_FILE) -Wl,-Map=$@.map -Wl,-cref -Wl,-gc-sections -specs=nano.specs
+LDFLAGS = $(CFLAGS) -fshort-enums -Wl,-nostdlib -Wl,-T,$(LD_FILE) -Wl,-T,$(LD_COMMON) -Wl,-Map=$@.map -Wl,-cref -Wl,-gc-sections -specs=nano.specs
 LIBS := -lgcc -lc
 
-LDFLAGS += -mthumb -mcpu=cortex-m4
+LDFLAGS += -mthumb $(MCU_FLAGS_$(MCU_SERIES))
```

Comment:
No need for this. CFLAGS has these values and is included above.

Reply:
I think I left this in back when I had mtune, which I think failed as a linker parameter